main.js

```diff
@@ -22,14 +27,31 @@ const pg = new Client({
 });
 pg.connect();
 
+app.use(flash());
+app.use(session({
+  secret : 'super secret key',
+  resave : false,
+  saveUninitialized : false
+}));
+app.use(passport.initialize());
+app.use(passport.session());
+
 app.get('/',function(req,res,next){
     let context = {};
+    if (req.user){
+      context.user = req.user
+    }
```

Comment:
You could simplify these repeated lines into a one-liner
`context.user = req.user || null`

Reply:
thanks, that seems like a better way to do it.

Reply:
done